shell/platform/linux/testing/mock_window.cc

```diff
@@ -0,0 +1,18 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/testing/mock_window.h"
+
+using namespace flutter::testing;
+
+static MockWindow* mock = nullptr;
```

Comment:
Up to you since the Linux embedder is mostly C but this file is mostly C++ due to testing code. Since we're in uncharted waters, stick to your preferences (so long as we're consistent with other Linux embedder code).

The way we'd do this in the Windows embedder would be:
```cc
namespace {
MockWindow* mock = nullptr;
}  // namespace
```

(Though we might not always be consistent there either)

Reply:
In general I think it's safer to stick with C conventions though I note that some C++ has crept into the tests. I have been considering converting those back where it makes sense. In general I think the code is more confusing with the mix.